src/inputs/input.jsx

```diff
@@ -10,16 +17,23 @@ const propTypes = {
     autoComplete: PropTypes.oneOf(['off', 'on']),
     autoFocus: PropTypes.bool,
     className: PropTypes.string,
+    /**
+     * A Input can be disabled.
+     */
```

Comment:
Nitpick: "**_An_** Input can be disabled." ;-)
Friendly neighborhood grammar police, at your service! 😁

Reply:
fixed.

Reply:
Looks good.  Resolving.